src/utils/i18n/i18nextLocize.ts

```diff
@@ -344,6 +345,15 @@ export const fetchTranslations = async (lang: string): Promise<I18nextResources>
  * @param i18nTranslations
  */
 const createI18nextLocizeInstance = (lang: string, i18nTranslations: I18nextResources): i18n => {
+  const langCustomerVariation = `${lang}-x-${process.env.NEXT_PUBLIC_CUSTOMER_REF}`;
+  const fallbackLang = lang === LANG_FR ? LANG_EN : LANG_FR;
+  i18nTranslations[langCustomerVariation] = cloneDeep(i18nTranslations[lang] || {});
+  // @ts-ignore
+  i18nTranslations[langCustomerVariation].common.examples.i18n.simpleTranslation = 'Overridden';
```

Comment:
Currently creating the variation (hardcoded). 

Will be replaced by an API fetch call later on to fetch the translations that have been overridden by the customer (`fr-x-customer1`, etc.)